src/lib/create/from-string.js

```diff
@@ -150,8 +150,11 @@ export function configFromRFC2822(config) {
             case 4: // Zone
                 timezone = timezones[match[5]];
                 break;
-            default: // UT or +/-9999
+            case 3: // UT
```

Comment:
Can you please order the case statements by length. Also you can add in comments some examples of each case on the first line.